src/CurrentlyOut/CurrentlyOutContainer.js

```diff
@@ -0,0 +1,106 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import { connect } from 'react-redux'
+
+import axios from 'axios'
+import { resetError, setLoaded } from '../redux/actions'
+import routePaths from '../routePaths'
+import Page from '../Components/Page'
+import Error from '../Error'
+import { ASC } from '../tablesorting/sortOrder'
+import SortableDataSource from '../tablesorting/SortableDataSource'
+import { lastNameComparator } from '../tablesorting/comparatorComposition'
+import CurrentlyOut from './CurrentlyOut'
+
+class CurrentlyOutContainer extends Component {
+  constructor(props) {
+    super(props)
+    this.state = {
+      currentlyOut: [],
+      location: {},
+    }
+  }
+
+  componentDidMount() {
+    this.getOffendersCurrentlyOut()
+  }
+
+  componentDidUpdate(previousProps) {
+    const { agencyId, history } = this.props
+
+    if (agencyId !== previousProps.agencyId) {
+      history.replace(routePaths.establishmentRoll)
+    }
+  }
+
+  async getOffendersCurrentlyOut() {
+    const {
+      handleError,
+      resetErrorDispatch,
+      setLoadedDispatch,
+      match: {
+        params: { livingUnitId },
+      },
+    } = this.props
+
+    try {
+      resetErrorDispatch()
+      setLoadedDispatch(false)
+      const response = await axios.get(`/api/movements/${livingUnitId}/currently-out`)
+      const { currentlyOut, location } = response.data
+      this.setState({ currentlyOut, location })
+      setLoadedDispatch(true)
+    } catch (error) {
+      handleError(error)
+    }
+  }
+
+  render() {
+    const { currentlyOut, location, loaded } = this.state
+    const { error } = this.props
+    return (
+      <Page title={`Currently out - ${location}`} error={error} loaded={loaded}>
```

Comment:
You can remove error and loaded because Page manages them locally via redux state

Reply:
Done